lib/matplotlib/sphinxext/plot_directive.py

```diff
@@ -316,47 +305,25 @@ def contains_doctest(text):
     return bool(m)
 
 
-@_api.deprecated("3.5", alternative="doctest.script_from_examples")
-def unescape_doctest(text):
-    """
-    Extract code from a piece of text, which contains either Python code
-    or doctests.
-    """
-    if not contains_doctest(text):
-        return text
-    code = ""
-    for line in text.split("\n"):
-        m = re.match(r'^\s*(>>>|\.\.\.) (.*)$', line)
-        if m:
-            code += m.group(2) + "\n"
-        elif line.strip():
-            code += "# " + line.strip() + "\n"
-        else:
-            code += "\n"
-    return code
-
-
-@_api.deprecated("3.5")
-def split_code_at_show(text):
+def _split_code_at_show(text, function_name):
```

Comment:
I don't think we need to mark any functions in tests as private. But it's not a big deal either way.

Reply:
this is is in a public module not the tests.